bank/handler.go

```diff
@@ -51,3 +59,96 @@ func handleMsgMultiSend(ctx sdk.Context, k Keeper, msg types.MsgMultiSend) sdk.R
 		Tags: tags,
 	}
 }
+
+// Handle MsgMintFeeToken
+func handleMsgTopup(ctx sdk.Context, k Keeper, msg types.MsgTopup, contractCaller helper.IContractCaller) sdk.Result {
+	if !k.GetSendEnabled(ctx) {
```

Comment:
Do we need `sendEnabled` or `sendDisabled` type functionality in `heimdall` ?

Reply:
Yes, we have. Governance module handles that for future use.